### DISCUSSION TITLE
Better squad management

### DISCUSSION THREAD

**Discussion by Simon125q** [dailydotdev/daily#1517] [Feature Requests]
It would be good to be able as a squad admin to not only allow all users to post or not but rather I would like to see also options to allow posting by user who passed some threshold when it comes to reputation points and to see option that every post must be first accepted by an administrator, also the ability to limit the amount of post the single user can make during the day to one squad would be good.

**Comment by idoshamun** [comment]
Hi @Simon125q,

We are working on admin approval for posts. Hopefully we can ship it in the upcoming weeks. Stay tuned
